x509-cert/tests/builder.rs

```diff
@@ -234,3 +235,20 @@ fn ecdsa_signer() -> ecdsa::SigningKey<NistP256> {
     let secret_key = p256::SecretKey::from_pkcs8_der(PKCS8_PRIVATE_KEY_DER).unwrap();
     ecdsa::SigningKey::from(secret_key)
 }
+
+#[test]
+fn certificate_request() {
+    let subject = Name::from_str("CN=service.domination.world").unwrap();
+
+    let signer = ecdsa_signer();
+    let mut builder = RequestBuilder::new(subject, &signer).expect("Create certificate request");
+    builder.add_extension(&SubjectAltName(vec![GeneralName::IpAddress(
+        OctetString::new(vec![192, 0, 2, 0]).unwrap(),
```

Comment:
note to self: get a conversion between https://doc.rust-lang.org/std/net/enum.IpAddr.html and `GeneralName`